hacking/report.py

```diff
@@ -5,11 +5,13 @@
 from __future__ import (absolute_import, print_function)
 
 import argparse
+import json
 import os
-import requests
 import sqlite3
 import sys
 
+from ansible.module_utils.urls import open_url
```

Comment:
Move this import down next to the other ansible import. That way the script will continue to work if you haven't sourced hacking/env-setup yet.

Reply:
done